src/components/Navbar/Navbar.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import Logo from "../logo/Logo";
+import { MenuItems } from "./MenuItems";
+import styles from "./Navbar.module.scss";
```

Comment:
If I'm correct the `CSS module` name convention is `component_name_without_uppercase.module.scss`.